doc/developer-notes.md

```diff
@@ -88,15 +88,22 @@ required when doing so would need changes to significant pieces of existing
 code.
   - Variable (including function arguments) and namespace names are all lowercase and may use `_` to
     separate words (snake_case).
-    - Class member variables have a `m_` prefix.
-    - Global variables have a `g_` prefix.
+    - Class member variables have a `m_` prefix, so it's obvious when variable
+      references in the code implicitly access the current class instance.
```

Comment:
MIght be better to be shorter "so it's obvious when code is accessing implicit instance member variables."

Reply:
Even pithier (here and below): `s/so it's obvious/to indicate/`

Reply:
I don't agree that that's the reason for using `m_` or `g_`; in my opinion the main benefit is that it avoids shadowing, so that parameters/local variables don't hide access to member variables or globals. In C++ any non-static member function implicitly has access to the current class instance, and you hopefully already know what function you're looking at.